tests/rpc/rpc_01-system_test.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package rpc
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSystemRPC(t *testing.T) {
+	if GOSSAMER_INTEGRATION_TEST_MODE != rpcSpec {
+		_, _ = fmt.Fprintln(os.Stdout, "Going to skip RPC spec tests")
+		return
+	}
+
+	testsCases := []struct {
+		description string
+		method      string
+		expected    interface{}
+		skip        bool
+	}{
+		{ //TODO
+			description: "test system_name",
+			method:      "system_name",
+			skip:        true,
+		},
+		{ //TODO
+			description: "test system_version",
+			method:      "system_version",
+			skip:        true,
+		},
+		{ //TODO
+			description: "test system_chain",
+			method:      "system_chain",
+			skip:        true,
+		},
+		{ //TODO
+			description: "test system_properties",
+			method:      "system_properties",
+			skip:        true,
+		},
+		{
+			description: "test system_health",
+			method:      "system_health",
+			expected: modules.SystemHealthResponse{
+				Health: common.Health{
+					Peers:           2,
```

Comment:
how are there two peers if you are only starting one node? maybe I am missing something

Reply:
That is not yet used, you can see that the assertion is currentlty not performed. I have a objective to run more peers and make this test pass as you can observe on: https://github.com/ChainSafe/gossamer/pull/793/files/f485eea41765dfb14c881ed0b610e2e6fc41fb3e#diff-e0ce41a76aeea782dcef103c35cec4bfR161